lib/config_files/grub2/default.rb

```diff
@@ -0,0 +1,176 @@
+require "config_files/base_model"
+require "config_files/augeas_parser"
+
+module ConfigFiles
+  module Grub2
+    # Represents grub configuration in /etc/default/grub
+    # Main features:
+    #
+    # - Do not overwrite files
+    # - When setting value first try to just change value if key already exists
+    # - When key is not set, then try to find commented out line with key and
+    #   replace it with real config
+    # - When even commented out code is not there, then append configuration
+    #   to the end of file
+    class Default < BaseModel
+      PARSER = AugeasParser.new("sysconfig.lns")
+      PATH = "/etc/default/grub"
+
+      def initialize(file_class: File)
```

Comment:
or load it your self and use memory file. But in general for yast I plan to create special tuner file_class that at first asks for SCR path.
Comparing to augeas itself it is improvement because augeas use only hardcoded path and root. This allows also usage over ssh/sockets/whatever.